rbac/management/group/view.py

```diff
@@ -360,7 +383,16 @@ def principals(self, request, uuid=None):
             if isinstance(resp, dict) and 'errors' in resp:
                 return Response(status=resp['status_code'], data=resp['errors'])
             output = GroupSerializer(resp)
-            return Response(status=status.HTTP_200_OK, data=output.data)
+            response = Response(status=status.HTTP_200_OK, data=output.data)
+        elif request.method == 'GET':
+            page = self.paginate_queryset(group.principals.all())
+            serializer = PrincipalSerializer(page, many=True)
+            usernames = serializer.data
```

Comment:
I think this is going to end up being an ordered dictionary here, which is what the `users` value in the [POST payload](https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/management/principal/proxy.py#L188) will be set to, and it looks like we actually [need a list](https://backoffice-proxy-insights-services.ext.us-east.aws.preprod.paas.redhat.com/docs/#/default/postUsers)?

Reply:
It has been returning an OrderedDict, but I wasn't sure if it needed that level of object specificity, but I can do a quick translation of the data into a list here.